tests/utils/addresses.ts

```diff
@@ -1,7 +1,6 @@
-import {
-  ROUTER_ADDRESS as MAINNET_ROUTER_ADDRESS,
-  PERMIT2_ADDRESS as MAINNET_PERMIT2_ADDRESS,
-} from '../../src/utils/constants'
+import { UNIVERSAL_ROUTER_ADDRESS, PERMIT2_ADDRESS as MAINNET_PERMIT2_ADDRESS } from '../../src/utils/constants'
+
+const MAINNET_ROUTER_ADDRESS = UNIVERSAL_ROUTER_ADDRESS(1)
 
 export const PERMIT2_ADDRESS =
   process.env.USE_MAINNET_DEPLOYMENT === 'true' ? MAINNET_PERMIT2_ADDRESS : '0x4a873bdd49f7f9cc0a5458416a12973fab208f8d'
```

Comment:
Should `0x4a873bdd49f7f9cc0a5458416a12973fab208f8d` and `0xe808c1cfeebb6cb36b537b82fa7c9eef31415a05` also be variables defined in `src/utils/constants`?